apps/dashboard/src/hooks/use-fetch-workflow-test-data.ts

```diff
@@ -3,11 +3,16 @@ import type { WorkflowTestDataResponseDto } from '@novu/shared';
 import { QueryKeys } from '@/utils/query-keys';
 import { getWorkflowTestData } from '@/api/workflows';
 import { useEnvironment } from '@/context/environment/hooks';
+import { getEncodedId, WORKFLOW_DIVIDER } from '@/utils/step';
 
-export const useFetchWorkflowTestData = ({ workflowSlug }: { workflowSlug?: string }) => {
+export const useFetchWorkflowTestData = ({ workflowSlug }: { workflowSlug: string }) => {
   const { currentEnvironment } = useEnvironment();
   const { data, isPending, error } = useQuery<WorkflowTestDataResponseDto>({
-    queryKey: [QueryKeys.fetchWorkflowTestData, currentEnvironment?._id, workflowSlug],
+    queryKey: [
```

Comment:
@desiprisg I think the test data page is better if we don't cache it at all. It's a separate tab, so I don't see the value of caching. We'd rather fetch data every time we visit the tab instead of caching and invalidating.